Tests/TuistLoaderTests/Models+ManifestMappers/Headers+ManifestMapperTests.swift

```diff
@@ -136,4 +136,127 @@ final class HeadersManifestMapperTests: TuistUnitTestCase {
             "Sources/project/D/D1.h",
         ].map { temporaryPath.appending(RelativePath($0)) })
     }
+    
+    func test_from_and_excluding() throws {
+        // Given
+        let temporaryPath = try self.temporaryPath()
+        let generatorPaths = GeneratorPaths(manifestDirectory: temporaryPath)
+        try createFiles([
+            "Sources/public/A1.h",
+            "Sources/public/A1.m",
+            "Sources/public/A2.h",
+            "Sources/public/A2.m",
+
+            "Sources/private/B1.h",
+            "Sources/private/B1.m",
+            "Sources/private/B2.h",
+            "Sources/private/B2.m",
+
+            "Sources/project/C1.h",
+            "Sources/project/C1.m",
+            "Sources/project/C2.h",
+            "Sources/project/C2.m",
+        ])
+
+        let manifest = ProjectDescription.Headers(public: .init(globs: ["Sources/public/**"], excluding: ["Sources/public/A2.h"]),
+                                                  private: .init(globs: ["Sources/private/**"], excluding: ["Sources/private/B1.h"]),
+                                                  project: .init(globs: ["Sources/project/**"], excluding: nil))
```

Comment:
(Same below) I would format it as:
```suggestion
        let manifest = ProjectDescription.Headers(
            public: .init(globs: ["Sources/public/**"], excluding: ["Sources/public/A2.h"]),
            private: .init(globs: ["Sources/private/**"], excluding: ["Sources/private/B1.h"]),
            project: .init(globs: ["Sources/project/**"], excluding: nil)
        )
```

Reply:
I've been pondering starting using Apple's `swift-format` which is much more opinionated. I agree with you here but I will also try to create a PR to change our formatters.

Reply:
fixed